test/mycms-tool/sign/test-file.sh

```diff
@@ -84,18 +84,14 @@ test_sanity() {
 		--cms-in="${CMS}" \
 		--data-in="${DATA}" \
 		--cert="${builddir}/gen/test2.crt" \
-		)" || die "sanity.verify.wrong"
-
-	[ "${out}" = "VERIFIED" ] && die "sanity.verify.wrong.result '${out}'"
+		)" && die "sanity.verify.wrong"
```

Comment:
these are not good as in test we should explicitly know that the verify was failed and there was not some technical issue before. otherwise a regression of parsing for example will pass the test.